MCprep_addon/world_tools.py

```diff
@@ -973,6 +974,126 @@ def world_time_update(self, context):
 	return
 
 
+class MCPREP_OT_render_panorama(bpy.types.Operator):
+	"""Render the Panorama images for a texture Pack"""
+	bl_idname = "mcprep.render_panorama"
+	bl_label = "Render Panorama"
+	bl_description = "Render Panorama for texture Pack"
+	bl_options = {'REGISTER', 'UNDO'}
+
+	panorama_resolution = bpy.props.IntProperty(
+		name="Render resolution",
+		description="The resolution of the output images",
+		default=512
+	)
+
+	save_path = bpy.props.StringProperty(
+		name="Output Folder",
+		description="The folder to output the renders to",
+		subtype="DIR_PATH",
+		default="/tmp\\"
```

Comment:
User should be forced to select, so I'd just assign an empty string `""` and then if the folder doesn't already exist in the execute function, `return {'CANCELLED'}` after doing a `self.report({"ERROR"}, "Directory {x} does not exist")` for the user.